databricks_cli/sdk/api_client.py

```diff
@@ -72,7 +73,10 @@ def __init__(self, user=None, password=None, host=None, token=None,
         self.session = requests.Session()
         self.session.mount('https://', TlsV1HttpAdapter())
 
-        self.url = "%s/api/%s" % (host, apiVersion)
+        parsed_url = urlparse(host)
+        scheme = parsed_url[0]
```

Comment:
Could we use the urlparse named return values instead of indices? e.g., parsed_url.scheme, parsed_url.hostname?